include/core/JamomaInterpolator.h

```diff
@@ -91,7 +91,7 @@ namespace Jamoma {
             @param x1		Sample value at prior integer index
 			@param x2		Sample value at next integer index
             @param x3		Unused sample value
-			@param delta 	Fractional location between x1 (delta=0) and x1 (delta=1)
+			@param delta 	Fractional location between x1 (delta=0) and x2 (delta=1)
 			@return			The interpolated value
```

Comment:
to be consistent with other places in our code, should it be x0 (current input) and x1 (previous input) ?